doc/source/libraries/xorbits_data/datasets.rst

```diff
@@ -0,0 +1,86 @@
+.. _10min_datasets:
+
+======================================
+10 minutes to :code:`xorbits.datasets`
+======================================
+
+.. currentmodule:: xorbits.datasets
+
+This is a short introduction to :code:`xorbits.datasets`.
+
+Datasets Creation
+-----------------
+
+You can create a dataset from Hugging Face datasets, just like the `datasets.load_dataset`.
+xorbits.datasets will load the dataset in parallel among multiple machines, while Hugging Face
+datasets can only load the dataset in parallel on one machine. Currently, only the dataset with
+multiple data files can be loaded in parallel, one data file will be one chunk.
+
+::
+
+    >>> import xorbits.datasets as xdatasets
+    >>> dataset = xdatasets.from_huggingface(
+    >>>     "mariosasko/test_multi_dir_dataset", split="train")
+    Dataset({
+        features: ['text'],
+        num_rows: 2
+    })
+
+Datasets Processing
+-------------------
+
+You can apply a function to the dataset, xorbits.datasets will parallel the apply operations.
+xorbits.datasets can process data in parallel at the granularity of chunks if the xorbits cluster
+has enough resources. If your dataset has too few chunks, then you can use the `rechunk()` to 
+improve concurrency.
+
+::
+
+    >>> import xorbits.datasets as xdatasets
+    >>> # The `rotten_tomatoes` dataset contains empty data files, so only one
+    >>> # chunk will be in the dataset.   
+    >>> dataset = xdatasets.from_huggingface("rotten_tomatoes", split="train")
+    >>> # Use rechunk() to improve the concurrency.
+    >>> dataset = dataset.rechunk(10)
+    >>> def add_prefix(example):
+    >>>     example["text"] = "Xorbits: " + example["text"]
+    >>>     return example
+    >>> # 10 processes applying `add_prefix` concurrently.
+    >>> dataset = dataset.map(add_prefix)
+    >>> # Currently, you have to execute() and fetch() to get all the dataset.
+    >>> dataset.execute()
```

Comment:
Is the `execute` required?

Reply:
Currently, the dataset can be auto executed, so the `execute()` is not required. But the meta is not being updated correctly, so the `fetch()` is required.

I will remove this `execute()` from doc.

The next PR is almost ready, then `execute()` and `fetch()` are not required.